deploy/bluemix/config/database.yml

```diff
@@ -9,3 +9,4 @@ postgres_url = service_config['credentials']['uri']
 %>
 production:
   url: "<%= postgres_url %>"
+  pool: 20 # Puma defaults to 16 threads per instance. +4 for leeway.
```

Comment:
Is this here intentionally? Seems unrelated to the rest of the work here re picture versions.

Reply:
Not related to the PR, but came in as part of the hotfixes I mentioned in Jira. I believe it was helpful with the database connection issue, but it didn't solve the problem entirely.